mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioScreen.tsx

```diff
@@ -597,14 +598,15 @@ export function PortfolioScreen ({ navigation }: Props): JSX.Element {
 function AssetSortRow (props: { isSorted: boolean, assetSortType: PortfolioSortType, modifiedDenominationCurrency: string, onPress: () => void }): JSX.Element {
   const highestCurrencyValue = translate('screens/PortfolioScreen', 'Highest value ({{modifiedDenominationCurrency}})', { modifiedDenominationCurrency: props.modifiedDenominationCurrency })
   const lowestCurrencyValue = translate('screens/PortfolioScreen', 'Lowest value ({{modifiedDenominationCurrency}})', { modifiedDenominationCurrency: props.modifiedDenominationCurrency })
+  const { language } = useLanguageContext()
   const getDisplayedSortText = useCallback((text: PortfolioSortType): string => {
     if (text === PortfolioSortType.HighestDenominationValue) {
       return highestCurrencyValue
     } else if (text === PortfolioSortType.LowestDenominationValue) {
       return lowestCurrencyValue
     }
     return text
-  }, [props.modifiedDenominationCurrency])
+  }, [props.modifiedDenominationCurrency, language])
```

Comment:
Can we check the sorting label again? It remains untranslated when i tested with the publish preview on iOS and android. 

In any case, i think we can remove `useCallback` of getDisplayedSortText, dont think that function is expensive in terms of computation

Reply:
https://user-images.githubusercontent.com/40191153/182805320-c1aa4307-9fd1-45b6-b2ec-861799f71f30.MOV

Reply:
Issue is only with english and German switch (This might be because of different bug). I will remove useCallback for now